rasa/shared/nlu/training_data/training_data.py

```diff
@@ -38,139 +38,37 @@
 TF_RECORD_KEY_SEPARATOR = "#"
 
 
-class TrainingData:
-    """Holds loaded intent and entity training data."""
+class TrainingDataForNLUPipeline:
```

Comment:
Would name it `NLUTrainingData` or `NLUPipelineTrainingData`

Reply:
I thought about it, but it contains not only NLU data now. `NLUPipelineTrainingData` sounds good